controllers/machinedeletion_controller.go

```diff
@@ -95,30 +140,54 @@ func (r *MachineDeletionReconciler) getRemediation(ctx context.Context, req ctrl
 	return remediation
```

Comment:
in line 136, there's:
`r.Log.Error(err, "error retrieving remediation %s in namespace %s: %v", req.Name, req.Namespace)`

there are 3 placholders (%s,%s,%v) but only 2 arguments, this will panic the process.